README.rst

```diff
@@ -93,6 +94,7 @@ UCI configuration options must go in ``/etc/config/openwisp``.
   (new device will be named after its mac address, to avoid having many new devices with the same name)
 - ``pre_reload_hook``: path to custom executable script, see `pre-reload-hook`_
 - ``post_reload_hook``: path to custom executable script, see `post-reload-hook`_
+- ``post_reload_delay``: delay in seconds to wait before the post-reload-hook and any configuration test, defaults to ``0``
```

Comment:
the default here is incorrect but I will fix it

Reply:
Oh, right